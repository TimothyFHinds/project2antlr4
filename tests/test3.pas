program test3;

var  
num1 : REAL;
num2 : REAL;

    //for-do loops
    //https://www.tutorialspoint.com/pascal/pascal_for_do_loop.htm
begin
    //basic for loop
    //for num1:= 1 to 10 do writeln(num1);

    for num2 := 10 to 20 do
    begin
        writeln('value of num2: ');
        writeln(num2);
        writeln();
    end;

    writeln();
    
    //for downto loop
    for num2 := 20 downto 10 do
    begin 
        writeln('value of num2: ');
        writeln(num2);
        writeln();
    end;
end.