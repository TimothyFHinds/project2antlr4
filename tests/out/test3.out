'value of num2: '
10.0

'value of num2: '
11.0

'value of num2: '
12.0

'value of num2: '
13.0

'value of num2: '
14.0

'value of num2: '
15.0

'value of num2: '
16.0

'value of num2: '
17.0

'value of num2: '
18.0

'value of num2: '
19.0

'value of num2: '
20.0


'value of num2: '
20.0

'value of num2: '
19.0

'value of num2: '
18.0

'value of num2: '
17.0

'value of num2: '
16.0

'value of num2: '
15.0

'value of num2: '
14.0

'value of num2: '
13.0

'value of num2: '
12.0

'value of num2: '
11.0

'value of num2: '
10.0

